From 9504e44db3be9c53f32936df472c75b063c9e06f Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Wed, 29 Mar 2017 10:25:04 +1100
Subject: [PATCH] Ensure sudo use on Solus

As su can't be used by default, lets use sudo instead.
---
 base/password.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/password.py b/base/password.py
index 443403f..ddc951b 100644
--- a/base/password.py
+++ b/base/password.py
@@ -55,6 +55,7 @@ AUTH_TYPES ={'mepis':'su',
              'gos':'sudo',
              'boss':'su',
              'lfs':'su',
+             'solus':'sudo',
              }


--
2.12.2
